less/global.less

```diff
@@ -12,3 +12,15 @@ h1, h2, h3, h4, h5 {
   font-family: 'Russo One', sans-serif;
 }
 
+html {
+  font-size: 62.5%;
+}
+
+img {
+  max-width: 100%;
```

Comment:
Your image in your desktop version is a bit stretched. Watch out for every little detail before submitting your final design.